UPGRADE-5.3.md

```diff
@@ -60,6 +60,11 @@ Messenger
  * Deprecated the `prefetch_count` parameter in the AMQP bridge, it has no effect and will be removed in Symfony 6.0
  * Deprecated the use of TLS option for Redis Bridge, use `rediss://127.0.0.1` instead of `redis://127.0.0.1?tls=1`
 
+Mime
+----
+
+ * Remove internal annotation to the method `getHeaderBody` and `getHeaderParameter`.
```

Comment:
[...] from the [...]

Reply:
Done :)

Reply:
there is a second to that should be replaced with from :)

Reply:
Or maybe reword it like this:

> Remove the internal annotation from the `getHeaderBody()` and `getHeaderParameter` methods of the `Headers` class.

Reply:
Changed